portafly/doc/ux_specs/product/edit_product.md

```diff
@@ -14,6 +14,10 @@
       * Name already exists on form submit
       * Name already exists on _onblur_
   * System name > text input [disabled]
+    * Inline help:
```

Comment:
And the idea is to do this on each form where there is a system name?

Reply:
@thomasmaas yes, it can probably just be the same message with an altered subject "Product", "Backend", "Application plan".

Reply:
@thomasmaas  I was thinking about offering a solution to that, meaning telling users that if they really need to change the system name they should then delete and re-create the Product / Backend / Application plan. Basically just to avoid users contacting support only for that. But that should actually include a series of cautionary measures, so maybe we can skip for now.

Reply:
Seems more like something for docs than for in-ui